js/gemini.js

```diff
@@ -219,11 +219,11 @@ module.exports = class gemini extends Exchange {
             status = 'canceled';
         }
         let price = this.safeFloat (order, 'price');
-        price = this.safeFloat (order, 'avg_execution_price', price);
```

Comment:
Hi! Thanks for the PR! 

I'd ask you to clarify the edits a bit, plz. Are you sure that this particular place is a bug? Because, the user would expect the actual price and cost rather than those submitted. We would set the price to the submitted price in cases there's no other option. 

There may be a bug in this line, namely, if the average price is defined and is not zero, then the order price should be set to the average price, otherwise it should be set to the submitted price. However in this particular line, the average price may indeed be defined and may have a price of zero, effectively overriding the submitted price. Is this the case? If so, then we might want to add a check for a zero average price in between. Your verbose output of the problematic order would help.

Thx for `fetchOrderOrders`!

Reply:
The raw output from Gemini for the open order is
`{            order_id:   "4120809855",
                                                 id:   "4120809855",
                                             symbol:   "zecusd",
                                           exchange:   "gemini",
                                avg_execution_price:   "0.00",
                                               side:   "buy",
                                               type:   "exchange limit",
                                          timestamp:   "1531846409",
                                        timestampms:    1531846409070,
                                            is_live:    true,
                                       is_cancelled:    false,
                                          is_hidden:    false,
                                         was_forced:    false,
                                    executed_amount:   "0",
                                   remaining_amount:   "0.5",
                                            options: [],
                                              price:   "150.45",
                                    original_amount:   "0.5"             }`
for an open order, and
`{'was_forced': False, 'executed_amount': '0.01', 'options': [], 'exchange': 'gemini', 'remaining_amount': '0', 'is_hidden': False, 'type': 'exchange limit', 'timestamp': '1512490113', 'id': '2406803076', 'side': 'sell', 'timestampms': 1512490113084, 'symbol': 'btcusd', 'avg_execution_price': '11718.65', 'price': '10000.00', 'is_live': False, 'original_amount': '0.01', 'order_id': '2406803076', 'is_cancelled': False}` for a closed order.
So we can override the price with the average execution price if it isn't 0, as you recommend.  In testing this, I found that `is_canceled` should be `is_cancelled`, so will update the PR with both of those

Reply:
All right please take another look and let me know.  Side note: the test closed order I used is from end of 2017, that BTC price is *wild* :laughing: